**KOMPUTASI STATISTIK MENGGUNAKAN SOFTWARE R**<br>
*Disusun oleh: Toriq Afanudin*

In [1]:
# Tipe data dalam pemrograman R
angka <- 42
print(angka)

[1] 42


In [2]:
text <- "Hello World"
desimal <- 73.33
boolean <- TRUE
print(text)
print(desimal)
print(boolean)

[1] "Hello World"
[1] 73.33
[1] TRUE


In [3]:
# Membuat vektor bilangan
angka <- c(1, 2, 3, 4, 5)

# Membuat vektor karakter
teks <- c("apel", "pisang", "jeruk")

# Membuat vektor logika
logika <- c(TRUE, FALSE, TRUE, TRUE, FALSE)

print(angka)
print(teks)
print(logika)

[1] 1 2 3 4 5
[1] "apel"   "pisang" "jeruk" 
[1]  TRUE FALSE  TRUE  TRUE FALSE


**PERHITUNGAN STATISTIK DESKRIPTIF**<br>
* Mean
* Median
* Standar Deviasi
* Summary

In [10]:
data <- c(82, 77, 69, 70, 80, 78, 65, 68, 73, 84, 78, 73, 88, 85, 65)
rata_rata <- mean(data)
print(data)
print(rata_rata)

 [1] 82 77 69 70 80 78 65 68 73 84 78 73 88 85 65
[1] 75.66667


In [11]:
nilaiMedian <- median(data)
print(nilaiMedian)

[1] 77


In [12]:
nilaiSD <- sd(data)
print(nilaiSD)

[1] 7.354947


In [13]:
maksimal <- max(data)
minimal <- min(data)

print(maksimal)
print(minimal)

[1] 88
[1] 65


In [15]:
keseluruhan <- summary(data)
print(keseluruhan)

   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
  65.00   69.50   77.00   75.67   81.00   88.00 


**UJI NORMALITAS KOLMOGOROV-SMIRNOV**

Jika belum memiliki package nortest, diharuskan menginstal terlebih dahulu dengan cara:<br>
**install.packages("nortest")**

In [16]:
library(nortest)
lillie.test(data)


	Lilliefors (Kolmogorov-Smirnov) normality test

data:  data
D = 0.11282, p-value = 0.8742


1. Rumusan Hipotesis<br>
$H_0$: Data berdistribusi normal<br>
$H_1$: Data tidak berdistribusi normal<br>
2. $\alpha$ = 0.05<br>
3. Kriteria pengujian<br>
Jika **p-value > $\alpha$** maka $H_0$ diterima, dengan kata lain data berdistribusi normal<br>
4. Kesimpulan<br>
Berdasarkan output didapat nilai **p-value = 0.8742** yang mengakibatkan **p-value > $\alpha$**<br>
sehingga dapat disimpulkan $H_0$ diterima, atau data berdistribusi normal.